README.Rmd

```diff
@@ -63,83 +63,80 @@ devtools::install_github("bcgov/bcgovr")
+
+
+**Step 2: Set up & populate local repository using `bcgovr`**
+
+*RStudio GUI users:* Open a fresh session of RStudio where you will open your New Project. By default, RStudio will create any New Project in the current working directory. You can check the working directory using `getwd()`. Set your preferred local working directory in the R console (e.g. `setwd(C:/_dev)`) or using the RStudio navigation tools (Session -> Set Working Directory -> Choose Directory...). Now open your New Project: Select Version Control from the Create Project menu -> Select Git -> Paste the URL of the github.com/bcgov repository from step 1 in Repository URL -> Click 'Create Project'. The local project name will be inherited from the remote repository, and your local project is now "connected" to your remote in [github.com/bcgov](github.com/bcgov).
+
+You can now run `analysis_skeleton()` to create a 'ready-to-go' set of folders & files in your local repository that encourage best practice in scientific computing and include all of the [required bcgov items](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md).
+
 
 ```{r, eval = FALSE}
-bcgovr::analysis_skeleton() 
+bcgovr::analysis_skeleton() ## directly calls the function from `bcgovr` library
+## OR
+library(bcgovr)
+analysis_skeleton()
 ```
 
-Frequently you may want to create an analysis skeleton outside of the working directory. This directory can be set using the `path` argument. Consider if you wanted to create an analysis skeleton named `bcgovr_analysis` in the `C:/_dev` directory. In this case, you would directly input the path which would also automatically create a `bcgovr_analysis` folder if it did not exist already:
+
+*R/RStudio Console Users:* You can set your local working directory in the R/RStudio Console (e.g. `setwd(C:/_dev)`) or you can use the `path` argument in `analysis_skeleton` to set the path where you want your local repository to exist. Supply the remote location using the `clone_git` argument.
 
 ```{r, eval = FALSE}
-bcgovr::analysis_skeleton(path = "C:\_dev\bcgovr_analysis") 
+bcgovr::analysis_skeleton(path = "C:\_dev\bcgovr_analysis", clone_git = "url of remote repository") 
 ```
 
-Regardless of the path, ultimately, `analysis_skeleton()` results in a [bcgov](https://github.com/bcgov) 'ready-to-go' directory for a new data analysis project:
+This should result in a [bcgov](https://github.com/bcgov) 'ready-to-go' local directory for a new data analysis project:
 
 ![](img/analysis_skeleton_output.PNG)
 
 
-##### Options
+**Step 3: Stage, Commit & Push local repository folders & files to remote repository** 
```

Comment:
Given the discussion on email I think this is a good level of 'git with RStudio', but I think we should probably link to 'Happy Git with R'...